app/assets/stylesheets/common/base/topic.scss

```diff
@@ -188,6 +186,33 @@
         margin-top: 0.25em;
       }
     }
+
+    @include viewport.from(md) {
+      &.can-edit-topic .fancy-title {
```

Comment:
Why is all of this stuff viewport-width-specific? How do people on smaller displays know how to edit the title?

Reply:
At smaller widths (tablets) this will be done via pencil icon. We will not have the hover effects border when a pencil icon is present.

Reply:
Why should that be dependent on the width of a browser?

If the intention is to target devices with no hover ability, then we should check that explicitly rather than guessing 'is mobile/tablet' based on viewport width.